### DISCUSSION TITLE
Documentation for API

### DISCUSSION THREAD

**Discussion by stephenfelix** [Yaskawa-Global/YMConnect#2] [Q&A] [answered]
I'm currently trying to understand the possibilities in interfacing with Yaskawa robot. YMConnect seems to be relatively new but I am interested to find out more about it's capabilities.

**Comment by gavanderhoorn** [comment]
https://developer.motoman.com/en/YMConnect?

**Comment by aseamans** [comment, marked as answer]
Hi @stephenfelix,
YMConnect is a new library that is used for controlling a Yaskawa Motoman robot. 
Some of its capabilities include...

- Reading and writing IO
- Reading and writing variables
- Motion control (YRC1000 and above)
- Reading faults
- File management

A full list of capabilities can be found by taking a look at the interfaces at [https://developer.motoman.com/en/YMConnect/Reference](url)

Thanks!
